mysticeti-core/src/block_store.rs

```diff
@@ -300,6 +300,28 @@ impl BlockStore {
         }
         parents.contains(earlier_block)
     }
+
+    // Returns all the ancestors of the later_block to the `earlier_round`, assuming that there is
+    // a path to them.
+    pub fn linked_to_round(
+        &self,
+        later_block: &Data<StatementBlock>,
+        earlier_round: RoundNumber,
+    ) -> Vec<Data<StatementBlock>> {
+        let mut parents = vec![later_block.clone()];
+        for r in (earlier_round..later_block.round()).rev() {
```

Comment:
Does it make sense to stop early this loop if the vector `parents` becomes empty? (or would it be a marginal improvement?)